wolfcrypt/src/random.c

```diff
@@ -2242,7 +2242,9 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         #include <randomNumGen.h>
 
         int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz) {
-            STATUS        status;
+            STATUS                status == ERROR;
```

Comment:
I believe this was meant to just be a single `=`, please correct typo. Otherwise looks good.

Reply:
@kaleb-himes, thanks! fixed typo.

Reply:
Resolved